Sources/SwiftDiscord/DiscordJSON.swift

```diff
@@ -104,13 +105,34 @@ extension JSONRepresentable {
     }
 }
 
+// For checking if a value should be included in JSON
+fileprivate protocol Emptyable {
+    var isEmpty: Bool { get }
+}
+extension Optional : Emptyable {
```

Comment:
I wish: `'fileprivate' modifier cannot be used with extensions that declare protocol conformances`
Maybe it'll happen in Swift 5